tpm/tpm_test.go

```diff
@@ -232,6 +232,33 @@ func TestResizeableSlice(t *testing.T) {
 	}
 }
 
+func TestNVDefineSpace(t *testing.T) {
+	rwc := openTPMOrSkip(t)
+	defer rwc.Close()
+	index := tpmutil.Handle(0x40000001)
+	permissions := []uint32{nvPerWriteSTClear}
+	length := uint32(54)
```

Comment:
Where does "54" come from in this example? Can it be calculated in the code instead of being hard coded?

Reply:
DefineSpace has a variable length, so i just typed 54. Can't be "calculated" because depends on usecase.

Reply:
I'm not sure I understand this response. Are you saying that 54 is arbitrary and doesn't correspond to any of the rest of the code? If so, then please add a comment to say that.